src/main/java/uk/gov/justice/laa/crime/cfecrime/cma/StubCma.java

```diff
@@ -14,7 +19,9 @@ public ValueList callCma(CmaRequest request) {
         DecimalValue adjustedLivingAllowance = new DecimalValue("adjusted_living_allowance", new BigDecimal("0.0"));
         DecimalValue adjustedIncome = new DecimalValue("adjusted_income", new BigDecimal("0.0"));
         StringValue result = new StringValue("result", AssessmentResult.PASS.name());
-        StringValue outcome = new StringValue("outcome", Outcome.Eligible.name());
+        MeansTestOutcome fullMeansTestOutcome = FullMeansTestOutcomeCalculator.getFullMeansTestOutcome(FullAssessmentResult.PASS, CaseType.COMMITAL, MagCourtOutcome.APPEAL_TO_CC);
+
+        StringValue outcome = new StringValue("outcome", fullMeansTestOutcome.getOutcome());
```

Comment:
Think this needs to be reverted. We will probably call this method from the controller